Context/BeverageContext.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+namespace ShalekKavy.Api.Context
+{
+    public class BeverageContext : DbContext
+    {
+        public BeverageContext(DbContextOptions<BeverageContext> options) : base(options) { }
```

Comment:
Where do you define the unique primary key for each beverage? Each item in the database needs to have a unique primary key.